lib/DDG/Spice/Sentencer.pm

```diff
@@ -0,0 +1,32 @@
+package DDG::Spice::Sentencer;
+
+# ABSTRACT: Output a random number of metaphor sentences.
+
+use strict;
+use DDG::Spice;
+
+# spice proxy_cache_valid => "418 1d";
+
+triggers startend => "metaphor", "metaphors", "metaphorpsum", "sentencer";
+
+spice to => 'http://metaphorpsum.com/sentences/$1?format=json';
+spice wrap_jsonp_callback => 1;
+
+handle remainder => sub {
+    #  print "SAAAAAMMMMMM!!!!!!";
+    #     print $_;
+    #     $_ = "$_";
+    #     print $_;
```

Comment:
This commented out code can be removed. While helpful for debugging, we probably don't want to check this in.